### ISSUE TITLE
create a hastag #transitdeserts and link it in the bottom or on the right side of page

### ISSUE THREAD

**Issue by JianweiCxyz** [JianweiCxyz/UIL_webpages#20]

**Comment by cbischak** [comment]
Need to discuss what we want the goals of this to be etc.

**Comment by JianweiCxyz** [comment]
Yeah, we might need to talk to Prof. Jiao about it.
I opened the previous issue because he mentioned it in an e-mail.

**Comment by cbischak** [comment]
Yeah I saw the email. This is a much more extensive undertaking in many ways than we can do by Monday. We could just create a sentence on the site for now like "If you liked this research consider tweeting about it with the hashtag transitdeserts." But I don't think this is really what we want to do however. Leave the issue open for now, but I'll mark it longer term.
